src/components/PageHeader/SlippageToleranceSetting.tsx

```diff
@@ -46,46 +46,46 @@ const predefinedValues = [
   { label: '0.5%', value: 0.5 },
   { label: '1%', value: 1 }
 ]
-
 const SlippageToleranceSettings = () => {
+  const TranslateString = useI18n()
   const [userSlippageTolerance, setUserslippageTolerance] = useUserSlippageTolerance()
   const [value, setValue] = useState(userSlippageTolerance / 100)
   const [error, setError] = useState<string | null>(null)
-
   const handleChange = (evt: React.ChangeEvent<HTMLInputElement>) => {
     const { value: inputValue } = evt.target
     setValue(parseFloat(inputValue))
   }
 
   // Updates local storage if value is valid
   useEffect(() => {
+    
     try {
       const rawValue = value * 100
       if (!Number.isNaN(rawValue) && rawValue > 0 && rawValue < MAX_SLIPPAGE) {
         setUserslippageTolerance(rawValue)
         setError(null)
       } else {
-        setError('Enter a valid slippage percentage')
+        setError(TranslateString(999, 'Enter a valid slippage percentage'))
       }
     } catch {
-      setError('Enter a valid slippage percentage')
+      setError(TranslateString(999, 'Enter a valid slippage percentage'))
     }
-  }, [value, setError, setUserslippageTolerance])
+  }, [value, setError, setUserslippageTolerance, TranslateString])
```

Comment:
@Chef-Chungus I think @RabbitDoge is right that it will cause this to run each re-render. He also noted that the translation context is not even used and we can probably move it out of being a hook and I agree.

As a temporary workaround I memoized the function and made the commit on this branch. This should allow it to be called inside an effect without causing issues.